### ISSUE TITLE
Notebook 1 errors

### ISSUE THREAD

**Issue by ljbennett62** [watson-developer-cloud/answer-retrieval#14]
- All of the code responses in the notebook were already populated upon opening the notebook
- It is not clear if one has to wait for the cluster to be available to continue past the checking the status of the cluster step?
- There are lots of pointers to the README. Why not make those links to the relevant section, or include the referenced information in the notebook?
- Second sentence under the Upload a Solr Cluster needs double check. I don’t think that’s an example?
- How does one know when Generate Training data step is completed?
- No information given about the format of the ground truth file. The notebook says it should be in the README, and it is there, but under a different name. The script mentioned works very VERY slowly, there should be some note of this somewhere.
- When attempting to train using the output of the bin/python/extract_stackexchange_dump.py script run on the cs.stackexchange.com dump:
  "Error encountered during training: Training Data Quality Standards Not Met: detected 2996 out of total 2999 queries as having no label variety. This exceeds the maximum allowed of 2249 (75%)."

**Comment by lkrishnamurthy** [comment]
Fixed issue 1, 2.  Notebooks do show an indication on the top right when the execution is complete. Also, with #15 fix the last issue seem to be resolved.
